tests/unit/api/test_request_models.py

```diff
@@ -11,9 +11,8 @@ def test_metadata_query_correct_version():
 
 
 def test_metadata_query_no_version():
-    with pytest.raises(ValidationError) as e:
+    with pytest.raises(ValidationError):
         MetadataQuery()
-    assert "('version',)" in str(e)
```

Comment:
Why remove this assert?

Reply:
The test initially failed because pydantic has changed the Validationerror message from v1 to v2.
I was not sure how necessary it was to verify the content of the actual error-message, and that it would be sufficient to verify that a validationerror is raised when there is no input.

But I can update the test if you think it is best? It can be:
assert "version" in str(e) 

Error messages in pydantic 1 and 2:
Pydantic v1: 
ExceptionInfo ValidationError(model='MetadataQuery', errors=[{'loc': ('version',), 'msg': 'field required', 'type': 'value_error.missing'}]) tblen=2>

Pydantic v2:
ExceptionInfo 1 validation error for MetadataQuery version Field required [type=missing, input_value={}, input_type=dict]

Reply:
> But I can update the test if you think it is best? It can be:
assert "version" in str(e)

This is perfect 👍🏻

Reply:
Updated 👍🏻